apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/common/exception/ExceptionCode.java

```diff
@@ -10,5 +10,8 @@
     PLACE_NOT_AVAILABLE,
     SPEAKER_UNAVAILABLE,
     PLACE_NOT_FOUND,
-    VALIDATION_FAILED, USER_NOT_FOUND
+    VALIDATION_FAILED,
+    USER_NOT_FOUND,
+    USER_ALREADY_SIGNED,
+    NO_FREE_PLACES,
 }
```

Comment:
Probably better name would be `ATTENDEES_LIMIT_EXCEDED`

Reply:
Done